File: app/src/main/java/com/myself/todo/Beans/Events.java
package com.myself.todo.Beans;

public class Events {

    private String evento;
    String descricao;
    private String UserID;
    private String data;
    private String id;
    private String status;


    public Events(String id, String evento, String descricao, String usuario, String data, String status) {
        this.evento = evento;
        this.UserID = usuario;
        this.data = data;
        this.id = id;
        this.descricao = descricao;
        this.status = status;
    }

    public Events() {
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getEvento() {
        return evento;
    }

    public void setEvento(String evento) {
        this.evento = evento;
    }

    public String getDescricao() {
        return descricao;
    }

    public void setDescricao(String descricao) {
        this.descricao = descricao;
    }

    public String getUserID() {
        return UserID;
    }

    public void setUserID(String usuario) {
        this.UserID = usuario;
    }

    public String getData() {
        return data;
    }

    public void setData(String data) {


        this.data = data;

    }

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }




}

File: app/src/main/java/com/myself/todo/Beans/Events.kt
package com.myself.todo.Beans


class Events {
    var evento: String? = null
    var descricao: String? = null
    var UserID: String? = null
    var data: String? = null
    var id: String? = null
    var favorite = false
    var tasks:ArrayList<Tarefas> = ArrayList()



    fun isAcreateEvent():Boolean{
      return  id.equals("createEvnt")
    }

    constructor()
    constructor(evento: String, descricao: String, UserID: String, data: String, id: String, favorite: Boolean,tarefas: ArrayList<Tarefas>) {
        this.evento = evento
        this.descricao = descricao
        this.UserID = UserID
        this.data = data
        this.id = id
        this.favorite = favorite
        this.tasks = tarefas
    }

    companion object{
        fun createEvent():Events{
            val e = Events()
            e.id = "createEvnt"
            return e
        }
    }
}